Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/.gitignore	(date 1607549675202)
@@ -4,5 +4,3 @@
 *.iml
 log
 *.patch
-
-
Index: src/main/java/ru/javawebinar/topjava/web/RootController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ru/javawebinar/topjava/web/RootController.java b/src/main/java/ru/javawebinar/topjava/web/RootController.java
--- a/src/main/java/ru/javawebinar/topjava/web/RootController.java	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/src/main/java/ru/javawebinar/topjava/web/RootController.java	(date 1607550071829)
@@ -23,7 +23,7 @@
         return "users";
     }
 
-    @GetMapping(value = "/login")
+    @GetMapping("/login")
     public String login() {
         return "login";
     }
Index: src/test/java/ru/javawebinar/topjava/web/RootControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/javawebinar/topjava/web/RootControllerTest.java b/src/test/java/ru/javawebinar/topjava/web/RootControllerTest.java
--- a/src/test/java/ru/javawebinar/topjava/web/RootControllerTest.java	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/src/test/java/ru/javawebinar/topjava/web/RootControllerTest.java	(date 1607549675188)
@@ -19,7 +19,6 @@
                 .andDo(print())
                 .andExpect(status().isOk())
                 .andExpect(view().name("users"))
-                .andExpect(forwardedUrl("/WEB-INF/jsp/users.jsp"))
                 .andExpect(forwardedUrl("/WEB-INF/jsp/users.jsp"));
     }
 
Index: src/main/java/ru/javawebinar/topjava/web/meal/MealUIController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ru/javawebinar/topjava/web/meal/MealUIController.java b/src/main/java/ru/javawebinar/topjava/web/meal/MealUIController.java
--- a/src/main/java/ru/javawebinar/topjava/web/meal/MealUIController.java	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/src/main/java/ru/javawebinar/topjava/web/meal/MealUIController.java	(date 1607550054718)
@@ -39,7 +39,7 @@
     }
 
     @Override
-    @GetMapping(value = "/filter")
+    @GetMapping("/filter")
     public List<MealTo> getBetween(
             @RequestParam @Nullable LocalDate startDate,
             @RequestParam @Nullable LocalTime startTime,
Index: src/test/java/ru/javawebinar/topjava/web/json/JsonUtilTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/javawebinar/topjava/web/json/JsonUtilTest.java b/src/test/java/ru/javawebinar/topjava/web/json/JsonUtilTest.java
--- a/src/test/java/ru/javawebinar/topjava/web/json/JsonUtilTest.java	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/src/test/java/ru/javawebinar/topjava/web/json/JsonUtilTest.java	(date 1607550097350)
@@ -11,7 +11,7 @@
 class JsonUtilTest {
 
     @Test
-    void readWriteValue() throws Exception {
+    void readWriteValue() {
         String json = JsonUtil.writeValue(adminMeal1);
         System.out.println(json);
         Meal meal = JsonUtil.readValue(json, Meal.class);
@@ -19,7 +19,7 @@
     }
 
     @Test
-    void readWriteValues() throws Exception {
+    void readWriteValues() {
         String json = JsonUtil.writeValue(meals);
         System.out.println(json);
         List<Meal> meals = JsonUtil.readValues(json, Meal.class);
Index: src/test/java/ru/javawebinar/topjava/UserTestData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/javawebinar/topjava/UserTestData.java b/src/test/java/ru/javawebinar/topjava/UserTestData.java
--- a/src/test/java/ru/javawebinar/topjava/UserTestData.java	(revision c8bd12c5ece2dfd18e5c3cb290244e32a8b40bb7)
+++ b/src/test/java/ru/javawebinar/topjava/UserTestData.java	(date 1607549675196)
@@ -13,10 +13,12 @@
 
 public class UserTestData {
     public static final TestMatcher<User> USER_MATCHER = TestMatcher.usingIgnoringFieldsComparator(User.class, "registered", "meals");
+
     public static TestMatcher<User> USER_WITH_MEALS_MATCHER =
             TestMatcher.usingAssertions(User.class,
+//     No need use ignoringAllOverriddenEquals, see https://assertj.github.io/doc/#breaking-changes
                     (a, e) -> assertThat(a).usingRecursiveComparison()
-                            .ignoringFields("registered", "meals.user").ignoringAllOverriddenEquals().isEqualTo(e),
+                            .ignoringFields("registered", "meals.user").isEqualTo(e),
                     (a, e) -> {
                         throw new UnsupportedOperationException();
                     });
