Index: src/test/java/ru/javawebinar/topjava/web/user/HerokuRestControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/javawebinar/topjava/web/user/HerokuRestControllerTest.java b/src/test/java/ru/javawebinar/topjava/web/user/HerokuRestControllerTest.java
--- a/src/test/java/ru/javawebinar/topjava/web/user/HerokuRestControllerTest.java	(revision e4b8fe0fd6cad2ecb69995ea93f6e2806ba2c51a)
+++ b/src/test/java/ru/javawebinar/topjava/web/user/HerokuRestControllerTest.java	(date 1608217772789)
@@ -7,7 +7,6 @@
 import org.springframework.http.MediaType;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
-import ru.javawebinar.topjava.UserTestData;
 import ru.javawebinar.topjava.util.exception.ErrorType;
 import ru.javawebinar.topjava.web.AbstractControllerTest;
 
@@ -57,7 +56,7 @@
         perform(MockMvcRequestBuilders.put(REST_URL + USER_ID)
                 .contentType(MediaType.APPLICATION_JSON)
                 .with(userHttpBasic(admin))
-                .content(UserTestData.jsonWithPassword(user, "password")))
+                .content(jsonWithPassword(user, "password")))
                 .andExpect(errorType(ErrorType.VALIDATION_ERROR))
                 .andExpect(detailMessage(EXCEPTION_UPDATE_RESTRICTION))
                 .andExpect(status().isUnprocessableEntity());
Index: src/test/java/ru/javawebinar/topjava/web/user/AdminRestControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/javawebinar/topjava/web/user/AdminRestControllerTest.java b/src/test/java/ru/javawebinar/topjava/web/user/AdminRestControllerTest.java
--- a/src/test/java/ru/javawebinar/topjava/web/user/AdminRestControllerTest.java	(revision e4b8fe0fd6cad2ecb69995ea93f6e2806ba2c51a)
+++ b/src/test/java/ru/javawebinar/topjava/web/user/AdminRestControllerTest.java	(date 1608217541387)
@@ -7,7 +7,6 @@
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.transaction.annotation.Propagation;
 import org.springframework.transaction.annotation.Transactional;
-import ru.javawebinar.topjava.UserTestData;
 import ru.javawebinar.topjava.model.Role;
 import ru.javawebinar.topjava.model.User;
 import ru.javawebinar.topjava.service.UserService;
@@ -92,7 +91,7 @@
 
     @Test
     void update() throws Exception {
-        User updated = UserTestData.getUpdated();
+        User updated = getUpdated();
         updated.setId(null);
         perform(MockMvcRequestBuilders.put(REST_URL + USER_ID)
                 .contentType(MediaType.APPLICATION_JSON)
@@ -106,7 +105,7 @@
 
     @Test
     void createWithLocation() throws Exception {
-        User newUser = UserTestData.getNew();
+        User newUser = getNew();
         ResultActions action = perform(MockMvcRequestBuilders.post(REST_URL)
                 .contentType(MediaType.APPLICATION_JSON)
                 .with(userHttpBasic(admin))
Index: config/messages/app_ru.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config/messages/app_ru.properties b/config/messages/app_ru.properties
--- a/config/messages/app_ru.properties	(revision e4b8fe0fd6cad2ecb69995ea93f6e2806ba2c51a)
+++ b/config/messages/app_ru.properties	(date 1608231812508)
@@ -44,8 +44,6 @@
 common.saved=Запись сохранена
 common.enabled=Запись активирована
 common.disabled=Запись деактивирована
-common.errorStatus=Статус ошибки
-common.appError=Ошибка приложения
 common.confirm=Вы уверены?
 common.save=Сохранить
 common.cancel=Отменить
@@ -66,6 +64,6 @@
 NotNull=[{0}] не должно быть пустым
 Email=Неверный формат [{0}]
 Range= [{0}] должно быть между {2} и {1}
-Length=Длинна [{0}] должена быть между {2} и {1}
+Length=Длинна [{0}] должна быть между {2} и {1}
 Size=Размер [{0}] должен быть между {2} и {1}
 SafeHtml=[{0}] может содержать небезопасный html контент
\ No newline at end of file
