Index: src/main/java/ru/javawebinar/topjava/web/meal/JspMealController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javawebinar/topjava/web/meal/JspMealController.java	(revision b5105efc31767d015b40449978e782378c95d01b)
+++ src/main/java/ru/javawebinar/topjava/web/meal/JspMealController.java	(date 1606689262011)
@@ -18,7 +18,7 @@
 import static ru.javawebinar.topjava.util.DateTimeUtil.parseLocalTime;
 
 @Controller
-@RequestMapping(value = "/meals")
+@RequestMapping("/meals")
 public class JspMealController extends AbstractMealController {
 
     @GetMapping("/delete")
Index: src/main/java/ru/javawebinar/topjava/web/user/AdminUIController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javawebinar/topjava/web/user/AdminUIController.java	(revision b5105efc31767d015b40449978e782378c95d01b)
+++ src/main/java/ru/javawebinar/topjava/web/user/AdminUIController.java	(date 1606727491108)
@@ -9,24 +9,24 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/admin/users")
+@RequestMapping(value = "/admin/users", produces = MediaType.APPLICATION_JSON_VALUE)
 public class AdminUIController extends AbstractUserController {
 
     @Override
-    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping
     public List<User> getAll() {
         return super.getAll();
     }
 
     @Override
     @DeleteMapping("/{id}")
-    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     public void delete(@PathVariable int id) {
         super.delete(id);
     }
 
     @PostMapping
-    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     public void create(@RequestParam String name,
                        @RequestParam String email,
                        @RequestParam String password) {
Index: src/main/java/ru/javawebinar/topjava/web/user/ProfileRestController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javawebinar/topjava/web/user/ProfileRestController.java	(revision b5105efc31767d015b40449978e782378c95d01b)
+++ src/main/java/ru/javawebinar/topjava/web/user/ProfileRestController.java	(date 1606727491103)
@@ -8,11 +8,11 @@
 import static ru.javawebinar.topjava.web.SecurityUtil.authUserId;
 
 @RestController
-@RequestMapping(ProfileRestController.REST_URL)
+@RequestMapping(value = ProfileRestController.REST_URL, produces = MediaType.APPLICATION_JSON_VALUE)
 public class ProfileRestController extends AbstractUserController {
     static final String REST_URL = "/rest/profile";
 
-    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping
     public User get() {
         return super.get(authUserId());
     }
@@ -29,7 +29,7 @@
         super.update(user, authUserId());
     }
 
-    @GetMapping(value = "/text")
+    @GetMapping("/text")
     public String testUTF() {
         return "Русский текст";
     }
Index: src/main/java/ru/javawebinar/topjava/web/user/AdminRestController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javawebinar/topjava/web/user/AdminRestController.java	(revision b5105efc31767d015b40449978e782378c95d01b)
+++ src/main/java/ru/javawebinar/topjava/web/user/AdminRestController.java	(date 1606689261999)
@@ -46,7 +46,7 @@
 
     @Override
     @PutMapping(value = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
-    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     public void update(@RequestBody User user, @PathVariable int id) {
         super.update(user, id);
     }
Index: src/main/java/ru/javawebinar/topjava/web/meal/MealRestController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javawebinar/topjava/web/meal/MealRestController.java	(revision b5105efc31767d015b40449978e782378c95d01b)
+++ src/main/java/ru/javawebinar/topjava/web/meal/MealRestController.java	(date 1606689262007)
@@ -27,7 +27,7 @@
 
     @Override
     @DeleteMapping("/{id}")
-    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     public void delete(@PathVariable int id) {
         super.delete(id);
     }
@@ -40,7 +40,7 @@
 
     @Override
     @PutMapping(value = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
-    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     public void update(@RequestBody Meal meal, @PathVariable int id) {
         super.update(meal, id);
     }
@@ -57,7 +57,7 @@
     }
 
     @Override
-    @GetMapping(value = "/filter")
+    @GetMapping("/filter")
     public List<MealTo> getBetween(
             @RequestParam @Nullable LocalDate startDate,
             @RequestParam @Nullable LocalTime startTime,
