Index: src/test/java/ru/javawebinar/topjava/UserTestData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/ru/javawebinar/topjava/UserTestData.java	(revision b619a4de1163e5d6fd30f725953f76e642de8bc9)
+++ src/test/java/ru/javawebinar/topjava/UserTestData.java	(date 1603295961966)
@@ -24,8 +24,12 @@
 
     public static User getUpdated() {
         User updated = new User(user);
+        updated.setEmail("update@gmail.com");
         updated.setName("UpdatedName");
         updated.setCaloriesPerDay(330);
+        updated.setPassword("newPass");
+        updated.setEnabled(false);
+        updated.setRoles(Collections.singletonList(Role.ADMIN));
         return updated;
     }
 
Index: src/test/java/ru/javawebinar/topjava/service/UserServiceTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/ru/javawebinar/topjava/service/UserServiceTest.java	(revision b619a4de1163e5d6fd30f725953f76e642de8bc9)
+++ src/test/java/ru/javawebinar/topjava/service/UserServiceTest.java	(date 1603109451314)
@@ -9,6 +9,7 @@
 import org.springframework.test.context.jdbc.Sql;
 import org.springframework.test.context.jdbc.SqlConfig;
 import org.springframework.test.context.junit4.SpringRunner;
+import ru.javawebinar.topjava.UserTestData;
 import ru.javawebinar.topjava.model.Role;
 import ru.javawebinar.topjava.model.User;
 import ru.javawebinar.topjava.util.exception.NotFoundException;
@@ -65,7 +66,7 @@
     @Test
     public void get() throws Exception {
         User user = service.get(USER_ID);
-        assertMatch(user, user);
+        assertMatch(user, UserTestData.user);
     }
 
     @Test
